index.js

```diff
@@ -283,7 +284,13 @@ Router.prototype.find = function find (method, path) {
 
     var node = currentNode.find(path[0], method)
     if (node === null) {
-      return getWildcardNode(wildcardNode, method, originalPath, pathLenWildcard)
+      node = currentNode.parametricBrother
```

Comment:
can this have two parametricBrothers?

Reply:
At the moment is not supported, we have already discussed that in https://github.com/delvedor/find-my-way/issues/33 :)